tests/prepare/artifact/data/test_require.sh

```diff
@@ -3,9 +3,23 @@
 
 rlJournalStart
     rlPhaseStartTest "Test artifact installation on Fedora"
+        # Verify tmt-artifact-shared repository is enabled
+        rlRun -s "dnf repo info tmt-artifact-shared"
+        rlAssertGrep "Status.*: enabled" $rlRun_LOG
+        rlLog "Repository tmt-artifact-shared is enabled and available"
+
+        # Verify make is installed
         rlRun "rpm -q make" 0 "Check that make is installed"
+
+        # Check that make comes from tmt-artifact-shared
         rlRun -s "dnf info --installed make"
```

Comment:
```suggestion
        # Check that make comes from tmt-artifact-shared
        rlRun "rpm -q make" 0 "Check that make is installed"
        rlRun -s "dnf info --installed make"
```
nitpick: these belong to the same series of checks